bokeh/models/tools.py

```diff
@@ -108,8 +108,8 @@
     'FreehandDrawTool',
     'HelpTool',
     'HoverTool',
-    'Inspection',
-    'Gesture',
+    'InspectTool',
+    'GestureTool',
```

Comment:
What's the justification for these breaking changes?

Reply:
For consistency with bokehjs and to make improve defaults' tests work without even more special cases. I can add aliases to aid backwards compatibility.